### PR TITLE
Fix coding style in NBitcoin - Part 1

### PR DESCRIPTION
**Pull request by quantumagi**
This PR fixes the coding style in the following files:
- BlockStore.cs
- Coins.cs
- CoinsView.cs
- PaymentACK.cs
- RawFormatter.cs
- RestClient.cs

### CODE REVIEW

File: src/NBitcoin/RPC/RawFormatter.cs [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+using System.IO;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
-using System.IO;
 
 namespace NBitcoin.RPC
 {
     abstract class RawFormatter
     {
+        public Network Network { get; set; }
```
Comment (fassadlr) [inline src/NBitcoin/RPC/RawFormatter.cs]: Is `Network` set anywhere else? Can we make this `readonly` and/or `protected`?

File: src/NBitcoin/Payment/PaymentACK.cs [hunk 2]
```diff
@@ -79,26 +71,52 @@ public void WriteTo(Stream output)
...
+        public readonly static string MediaType = "application/bitcoin-payment";
+
+        private readonly List<PaymentOutput> refundTo = new List<PaymentOutput>();
+        private readonly List<Transaction> transactions = new List<Transaction>();
+
+        public string Memo { get; set; }
+        public byte[] MerchantData { get; set; }
+        public Uri ImplicitPaymentUrl { get; set; }
+
+        public List<Transaction> Transactions
```
Comment (fassadlr) [inline src/NBitcoin/Payment/PaymentACK.cs]: Ah can't we just single this?

`public List<Transaction> Transactions { get { return this.transactions; } }`

File: src/NBitcoin/Payment/PaymentACK.cs [hunk 3]
```diff
@@ -79,26 +71,52 @@ public void WriteTo(Stream output)
...
+        {
+            get
+            {
+                return this.transactions;
+            }
+        }
+
+        public List<PaymentOutput> RefundTo
```
Comment (fassadlr) [inline src/NBitcoin/Payment/PaymentACK.cs]: Same here

File: src/NBitcoin/BitcoinCore/Coins.cs [hunk 4]
```diff
@@ -5,132 +5,138 @@ namespace NBitcoin.BitcoinCore
...
             }
             set
             {
-                fCoinStake = value;
+                this.fCoinStake = value;
             }
         }
 
-        uint nTime;
         public uint Time
```
Comment (fassadlr) [inline src/NBitcoin/BitcoinCore/Coins.cs]: Can you maybe collapse some of the extra lines here?

File: src/NBitcoin/BitcoinCore/Coins.cs [hunk 5]
```diff
@@ -5,132 +5,138 @@ namespace NBitcoin.BitcoinCore
...
-        bool fCoinStake;
+        private bool fCoinStake;
+        private uint nTime;
+        private uint nHeight;
+        private uint nVersion;
+
+        // Determines whether transaction is a coinbase.
+        public bool CoinBase { get; set; }
+
         public bool CoinStake
```
Comment (fassadlr) [inline src/NBitcoin/BitcoinCore/Coins.cs]: Can you maybe collapse some of the extra lines here?

File: src/NBitcoin/RPC/RawFormatter.cs (hunk 1 above)
Comment (quantumagi) [reply src/NBitcoin/RPC/RawFormatter.cs]: Transaction.cs is setting it

File: src/NBitcoin/Payment/PaymentACK.cs (hunk 3 above)
Comment (quantumagi) [reply src/NBitcoin/Payment/PaymentACK.cs]: Changed.

File: src/NBitcoin/Payment/PaymentACK.cs (hunk 2 above)
Comment (quantumagi) [reply src/NBitcoin/Payment/PaymentACK.cs]: Changed.

File: src/NBitcoin/BitcoinCore/BlockStore.cs [hunk 6]
```diff
@@ -99,108 +108,113 @@ public void SynchronizeStratisChain(ChainBase chain)
...
                         inChain.Add(block.Key);
                         last = chainedHeader;
                     }
                 }
-                foreach (var item in toRemove)
+
+                foreach (uint256 item in toRemove)
                     blocks.Remove(item);
+
                 if (toRemove.Count == 0)
```
[the remark this answers is not in the document]
Comment (quantumagi) [reply src/NBitcoin/BitcoinCore/BlockStore.cs]: Done.

File: src/NBitcoin/BitcoinCore/BlockStore.cs [hunk 7]
```diff
@@ -3,92 +3,101 @@
...
+            foreach (Block block in this.Enumerate(false).Select(b => b.Item))
             {
-                var hash = block.GetHash();
+                uint256 hash = block.GetHash();
                 blocks.TryAdd(hash, block);
             }
-            List<uint256> toRemove = new List<uint256>();
+
+            var toRemove = new List<uint256>();
             while (blocks.Count != 0)
```
[the remark this answers is not in the document]
Comment (quantumagi) [reply src/NBitcoin/BitcoinCore/BlockStore.cs]: Done.

File: src/NBitcoin/BitcoinCore/BlockStore.cs [hunk 8]
```diff
@@ -3,92 +3,101 @@
...
                         inChain.Add(header.Key);
                     }
                 }
-                foreach(var item in toRemove)
+
+                foreach (uint256 item in toRemove)
                     headers.Remove(item);
-                if(toRemove.Count == 0)
+
+                if (toRemove.Count == 0)
```
[the remark this answers is not in the document]
Comment (quantumagi) [reply src/NBitcoin/BitcoinCore/BlockStore.cs]: Done.

File: src/NBitcoin/BitcoinCore/BlockStore.cs [hunk 9]
```diff
@@ -3,92 +3,101 @@
...
-                var hash = header.GetHash();
+                uint256 hash = header.GetHash();
                 headers.Add(hash, header);
             }
-            List<uint256> toRemove = new List<uint256>();
-            while(headers.Count != 0)
+
+            var toRemove = new List<uint256>();
+
+            while (headers.Count != 0)
```
[the remark this answers is not in the document]
Comment (quantumagi) [reply src/NBitcoin/BitcoinCore/BlockStore.cs]: Done.

File: src/NBitcoin/BitcoinCore/Coins.cs (hunk 4 above)
Comment (quantumagi) [reply src/NBitcoin/BitcoinCore/Coins.cs]: Not sure what you mean. Only one blank line max to improve readability.

File: src/NBitcoin/BitcoinCore/Coins.cs (hunk 5 above)
Comment (quantumagi) [reply src/NBitcoin/BitcoinCore/Coins.cs]: Not sure what you mean. Only one blank line max to improve readability.

**Comment by quantumagi** [comment]
Thank you @fassadlr. I've made most of the changes that you have suggested.

**Pull request merged**